Object.h

```diff
@@ -33,6 +33,7 @@ typedef void(*Object_Delete)(Object*);
 #define Class(class_)                 ((const ObjectClass*)(&(class_ ## _class)))
 
 #define AllocThis(class_) (class_*)   xMalloc(sizeof(class_)); Object_setClass(this, Class(class_))
+#define CallocThis(class_) (class_*)   xCalloc(sizeof(class_), 1); Object_setClass(this, Class(class_))
```

Comment:
Not a fan of this. Also mixes up order of arguments for `xCalloc`.

Reply:
I made this function to avoid undefined memory data during the allocation.
But I didn't notice,  `AllocThis` is not `MallocThis`. So adding a function/macro called `CallocThis` can be a non-sense.
So I'll make a function that initializes the backtrace panel and the frame (like `BacktracePanel_init`).